keyboards/bacca70/readme.md

```diff
@@ -0,0 +1,15 @@
+# Bacca70
+
+![Bacca70](https://imgur.com/TQMALjwh.png)
+
+The Bacca70 is a winkeyless (WKL) 60% layout that includes a F-row. It's classy, hefty and it was designed by Chewwy (of MechsOnDeck) and BlindAssassin.
+
+* Keyboard Maintainer: [keebnewb](https://github.com/thompson-ele)  
+* Hardware Supported: Bacca70 PCB and the ortholinear variant the DeBaccaBean PCB
+* Hardware Availability: [3DKeebs](https://3dkeebs.com/)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
@drashna updated 👍